RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -23,17 +23,14 @@ import RevenueCat
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
 @MainActor class CustomerCenterViewModel: ObservableObject {
-    // We fail open.
+
     private static let defaultAppIsLatestVersion = true
 
     typealias CurrentVersionFetcher = () -> String?
 
     private lazy var currentAppVersion: String? = currentVersionFetcher()
-
     @Published
-    private(set) var hasActiveProducts: Bool = false
-    @Published
-    private(set) var hasAppleEntitlement: Bool = false
+    private(set) var purchaseInformation: PurchaseInformation?
```

Comment:
Big improvement is holding the `purchaseInformation` here instead of in `ManageSubscriptionsViewModel`. That way most of the logic happens in this viewmodel and we don't need to get customer info anywhere else